hackathon_site/dashboard/frontend/src/pages/Inventory/Inventory.test.tsx

```diff
@@ -6,6 +6,7 @@ import Inventory from "pages/Inventory/Inventory";
 import { mockCategories, mockHardware } from "testing/mockData";
 
 import { get } from "api/api";
+import { within } from "@testing-library/react";
```

Comment:
Minor, can you import this from `testing/utils`? That module re-exports everything from RTL (which they [recommend](https://testing-library.com/docs/react-testing-library/setup/#custom-render))